src/components/pages/ViewAllMeetings/MeetingInfo.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const MeetingInfo = props => {
+  const { info } = props;
+
+  return (
+    <div>
+      <p>{info.meeting_id}</p>
```

Comment:
This is a nice summary of properties for those who are not familiar with the info.  Thanks for including that at this early stage.